src/main/java/com/telefonica/iot/cygnus/backends/http/HttpBackend.java

```diff
@@ -269,7 +269,13 @@ private JsonResponse createJsonResponse(HttpResponse httpRes) throws Exception {
             
             if (entity != null) {
                 BufferedReader reader = new BufferedReader(new InputStreamReader(httpRes.getEntity().getContent()));
-                String res = reader.readLine();
+                String res = "";
+                String line;
+
+                while ((line = reader.readLine()) != null) {
+                    res += line;
+                } // while
+
                 LOGGER.debug("response payload: " + res);
```

Comment:
It is not part of this issue, but we can do it right now: "Http response payload: "